src/main/java/de/tum/in/www1/artemis/domain/ShortAnswerQuestion.java

```diff
@@ -238,7 +238,7 @@ public ShortAnswerSpot findSpotById (Long spotId) {
     public void undoUnallowedChanges(QuizQuestion originalQuizQuestion) {
         if (originalQuizQuestion instanceof ShortAnswerQuestion) {
             ShortAnswerQuestion shortAnswerOriginalQuestion = (ShortAnswerQuestion) originalQuizQuestion;
-            undoUnallowedSolutionChanges(shortAnswerOriginalQuestion);
+            //undoUnallowedSolutionChanges(shortAnswerOriginalQuestion);
```

Comment:
please remove out commented code, if it is unnecessary

Reply:
done